eng/performance/sdk_scenarios.proj

```diff
@@ -103,5 +103,15 @@
       <Timeout>4:00</Timeout>
     </HelixWorkItem>
   </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetsWindows)' == 'true'">
+    <HelixWorkItem Include="SDK Inner Loop Console">
```

Comment:
nit: "SDK" sort of implies ownership in the SDK team. Is that appropriate here?

Reply:
Yeah, this was just a copy paste. I will change this.